README.md

```diff
@@ -149,6 +152,35 @@ netable.request(GetCatImages()) { result in
 }
 ```
 
+#### Canceling A Request
+
+You're able to easily cancel a request using `.cancel()`, which you can see in action in the [AuthNetworkService](https://github.com/steamclock/netable/blob/aeo/86-new-example-project/Netable/Example/Services/AuthNetworkService.swift#L82) within the Example Project.
```

Comment:
It might make sense that you need to change your request around a little to retain a reference to it, this might be worth a small example code block

Reply:
Sorry, I can't quite make sense of this! The request its pointing to uses the cancel by default, unless someone chooses to comment it out. What do you mean by "retain a reference to it"? I can definitely provide a small code block, I just want to clarify this point!

Reply:
Woof I'm sorry, I did not proofread that comment well enough 😬 

What I was _trying_ to say is that to use `cancel()`, you need to retain a reference to your Task:

```
 let createRequest = Task {
               let result = try await netable.request()
}

createRequest.cancel()
```

Which isn't normal, so we should probably put a little code sample in